common/list/_lists.scss

```diff
@@ -14,6 +14,9 @@
  * - views/_screen-lg-min
  * - views/_screen-xs-max
  */
+ul,ol {
+	margin-top: 0px;
+}
```

Comment:
That was not in the wet-boew SCSS

Why or please remove. Thanks

```suggestion
```

Reply:
Done